luigi/contrib/s3.py

```diff
@@ -500,17 +501,31 @@ def _get_s3_config(self, key=None):
 
         return section_only
 
-    def _path_to_bucket_and_key(self, path):
+    @staticmethod
+    def _path_to_bucket_and_key(path):
         (scheme, netloc, path, query, fragment) = urlsplit(path)
         path_without_initial_slash = path[1:]
         return netloc, path_without_initial_slash
 
-    def _is_root(self, key):
+    @staticmethod
+    def _is_root(key):
         return (len(key) == 0) or (key == '/')
 
-    def _add_path_delimiter(self, key):
+    @staticmethod
+    def _add_path_delimiter(key):
         return key if key[-1:] == '/' or key == '' else key + '/'
 
+    @staticmethod
+    def _check_deprecated_argument(arguments):
```

Comment:
Wouldn't it be best to make this `_check_deprecated_argument(**kwargs)`? to keep with Python practice